src/plays/image-compressor/ImageCompressor.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from "react";
+import PlayHeader from "common/playlists/PlayHeader";
+import imageCompression from "browser-image-compression";
+import CircularProgress from "@mui/material/CircularProgress";
+import "./styles.css";
+
+function ImageCompressor(props) {
+  // Your Code Start below.
+
+  const [selectedFile, setSelectedFile] = useState(null);
+  const [selectedFileSize, setSelectedFileSize] = useState(null);
+  const [compressing, setCompressing] = useState(false);
+  const [compressedFile, setCompressedFile] = useState(null);
+  const [compressedFileSize, setCompressedFileSize] = useState(null);
+
+  const handleImageUpload = async (event) => {
+    const imageFile = event.target.files[0];
+    if (!imageFile) {
+      return;
+    }
+    setSelectedFile(imageFile);
+    setSelectedFileSize((imageFile.size / 1024 / 1024).toFixed(2));
+    setCompressing(true);
+    const options = {
+      maxSizeMB: 1,
+      maxWidthOrHeight: 1920,
+      useWebWorker: true,
+    };
+    try {
+      const compressedFile = await imageCompression(imageFile, options);
+      setCompressedFile(compressedFile);
+      setCompressedFileSize((compressedFile.size / 1024 / 1024).toFixed(2));
+      setCompressing(false);
+    } catch (error) {
+      console.log(error);
```

Comment:
Can we avoid adding a console log and show some kind of error message to the user?